_posts/2020-07-06-private-bastions.md

```diff
@@ -0,0 +1,180 @@
+---
+layout: post
+title: 'Private Bastion - I'll do what you want me to do'
+date: 2020-07-06
+tags: aws ssm vpc ssh ec2
+author: Paul Symons
+---
+
+This post and its title are inspired by Tina Turner and her 1984 album "Private Dancer". Themes of fortitude, defiance and resiliency echo throughout her life story and her works, yet it is her belief that we can find a better way, that resonates most strongly right now.
+
+> *May you always find a way to turn poison into medicine* <br/><br/>Tina Turner, 2019
+
+# Private Bastions - Using VPC Endpoints to teleport into your VPC
+
+[Bastion Hosts](https://en.wikipedia.org/wiki/Bastion_host) are a fairly well known and aged concept - that of a server that acts as a solitary, publicly accessible network access point, placing authorised users within the perimeter of an otherwise private network.
```

Comment:
Do we need to mention that this may be a jumpbox in the Aus context?

Reply:
I think googling can be left to the imagination of the reader